Makefile

```
@@ -0,0 +1,68 @@
+# Ontology-metadata ontology Makefile
+# Jie Zheng
+#
+# This Makefile is used to build artifacts
+# for the IAO ontology metadata.
+#
+
+### Configuration
+#
+# prologue:
+# <http://clarkgrubb.com/makefile-style-guide#toc2>
+
+MAKEFLAGS += --warn-undefined-variables
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := all
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+### Definitions
+
+SHELL   := /bin/bash
+OBO     := http://purl.obolibrary.org/obo
+IAO  	:= $(OBO)/IAO_
+TODAY   := $(shell date +%Y-%m-%d)
+
+### Directories
+#
+# This is a temporary place to put things.
+build:
+	mkdir -p $@
+
+
+### ROBOT
+#
+# We use the official development version of ROBOT
+build/robot.jar: | build
+	curl -L -o $@ https://github.com/ontodev/robot/releases/download/v1.1.0/robot.jar
+
+ROBOT := java -jar build/robot.jar
+
+### Build
+#
+# Here we create a standalone OWL file appropriate for release.
+# This involves annotating.
+
+build/ontology-metadata_versioned.owl: src/ontology/ontology-metadata.owl | build/robot.jar build
+	$(ROBOT) annotate \
```

There's nothing to merge here, but the release file should be reasoned:

```suggestion
	$(ROBOT) reason \
	--input $< \
	--reasoner HermiT \
	annotate \
```